homeassistant/components/asuswrt/router.py

```diff
@@ -354,6 +372,22 @@ async def _update_unpolled_sensors(self) -> None:
             if self._sensors_data_handler.update_device_count(self._connected_devices):
                 await coordinator.async_refresh()
 
+    async def _get_available_temperature_sensors(self):
+        """Check which temperature information is available on the router."""
+        try:
+            availability = await self._api.async_find_temperature_commands()
+        except (OSError, ValueError):
+            return []
+        except Exception as exc:  # pylint: disable=broad-except
+            _LOGGER.warning(
+                "Failed checking temperature sensor availability for ASUS router %s. Exception: %s",
```

Comment:
I would be more for a `debug`, probably does not make sense warn users for this. But you can leave as is and wait for member team opinion.

Reply:
I debated about it too. Figured it's an unexpected exception - so I better not hide it in debug log.

(btw - thanks a lot for your feedback)

Reply:
now logging to 'debug'